source/includes/v4/_events.md

```diff
@@ -576,6 +576,7 @@ or scale prices with following attributes:
 - group (boolean), whether the price is for the whole group or per participant (per seat)
 - optional (boolean), whether the price is a choice or not
 - price_cents (integer) price in EUR cents
+- vat_pct (float), the pricing tax rate
```

Comment:
@kamil98 can we update the example response as well?

Reply:
do we even have an example response for scale prices?

Reply:
no, you are right.